l10n_ch_hr_payroll/migrations/8.0.1.0.09/pre-update-ir_model_data.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
```

Comment:
As the version in `__openerp__.py` is `1.0.8`, the folder should be `8.0.1.0.8` and not `8.0.1.0.09` (as well as the header's comment). Now the migration script is not loaded. Could you correct that?